internal/workflow_testsuite.go

```diff
@@ -318,6 +318,12 @@ func (e *TestWorkflowEnvironment) SetContinueAsNewSuggested(suggest bool) {
 	e.impl.setContinueAsNewSuggested(suggest)
 }
 
+// SetContinuedExecutionID sets the value that is returned from
+// GetInfo(ctx).ContinuedExecutionRunID
+func (e *TestWorkflowEnvironment) SetContinuedAsNewRunID(rid string) {
+	e.impl.setContinuedExecutionID(rid)
+}
```

Comment:
```suggestion
func (e *TestWorkflowEnvironment) SetContinuedExecutionRunID(rid string) {
	e.impl.setContinuedExecutionRunID(rid)
}
```

Let's match the field name.

Reply:
ho shoot, my bad, it was my intent but I mixed things up.